addons/web/static/src/scss/primary_variables.scss

```diff
@@ -7,7 +7,8 @@
 $o-webclient-color-scheme: bright !default;
 
 // Font sizes
-$o-root-font-size: 14px !default;
+$o-root-font-base: 14 !default;
```

Comment:
Where it's used?

Reply:
W'oops ! Forgot to remove this line, fixed !